packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -471,4 +471,17 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
 
     Terra.it.matchesScreenshot('date selected');
   });
+
+  describe('Remounts component', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-remount');
+    });
+
+    it('selects a date from date picker', () => {
+      browser.keys('ArrowDown');
+      browser.keys('Enter');
```

Comment:
don't we need to click on calendar button to select date here..?

Reply:
Good catch! Updated here: https://github.com/cerner/terra-framework/pull/1091/commits/2898d118261634c3eafec4ddc7b47c0d6ea5a664